01_fib/shahriys/main_test.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buffib bytes.Buffer
+	outfib = &buffib
+	main()
+	expected :=
+		`1
+1
+2
+3
+5
+8
+13
+1
+-1
+2
+-3
+5
+-8
+13
+`
+	actual := buffib.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
```

Comment:
It can be helpful to print the actual and expected when a test fails. It makes it actionable.

Reply:
yap good suggestion. I actually added while checking. Later removed it

Reply:
Can you re-add it?